onfido.gemspec

```diff
@@ -1,34 +1,43 @@
-# frozen_string_literal: true
-
-lib = File.expand_path('lib', __dir__)
-$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'onfido/version'
-
-Gem::Specification.new do |spec|
-  spec.name          = 'onfido'
-  spec.version       = Onfido::VERSION
-  spec.authors       = ['Onfido']
-  spec.email         = ['engineering@onfido.com']
-  spec.summary       = 'A wrapper for Onfido API'
-  spec.description   = "A thin wrapper for Onfido's API. This gem only supports "\
-                       "v3 of the Onfido API. Refer to Onfido's "\
-                       'API documentation for details of the expected '\
-                       'requests and responses.'
-  spec.homepage      = 'http://github.com/onfido/onfido-ruby'
-  spec.license       = 'MIT'
-
-  spec.files         = `git ls-files -z`.split("\x0")
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
-  spec.require_paths = ['lib']
-  spec.required_ruby_version = '>= 2.4.0'
-
-  spec.add_development_dependency 'rspec', '~> 3.1'
-  spec.add_development_dependency 'rspec-its', '~> 1.2'
-  spec.add_development_dependency 'rubocop', '~> 1.11'
-  spec.add_development_dependency 'sinatra', '~> 1.4'
-  spec.add_development_dependency 'webmock', '~> 3.0'
-
-  spec.add_dependency 'rack', '>= 1.6.0'
-  spec.add_dependency 'rest-client', '~> 2.0'
+# -*- encoding: utf-8 -*-
+
+=begin
+#Onfido API v3.6
+
+#The Onfido API (v3.6)
+
+The version of the OpenAPI document: v3.6
+
+Generated by: https://openapi-generator.tech
+Generator version: 7.5.0
+
+=end
+
+$:.push File.expand_path("../lib", __FILE__)
+require "onfido/version"
+
+Gem::Specification.new do |s|
+  s.name        = "onfido"
+  s.version     = Onfido::VERSION
+  s.platform    = Gem::Platform::RUBY
+  s.authors     = ["OpenAPI-Generator"]
+  s.email       = [""]
+  s.homepage    = "https://openapi-generator.tech"
+  s.summary     = "Onfido API v3.6 Ruby Gem"
+  s.description = "The Onfido API (v3.6)"
+  s.license     = "Unlicense"
```

Comment:
We should customize these fields (in generator configuration or via templating) before the actual delivery.

Reply:
done on generator side.